unblob/cli.py

```diff
@@ -6,13 +6,25 @@
 import click
 from structlog import get_logger
 
+from .dependencies import get_dependencies, show_external_dependencies
 from .logging import configure_logger, noformat
 from .processing import DEFAULT_DEPTH, process_file
 from .state import exit_code_var
 
 logger = get_logger()
 
 
+def override_cli_help_text(f):
+    commands = get_dependencies()
+    f.__doc__ = """A tool for getting information out of any kind of binary blob.
```

Comment:
`click.command()` has a help kwarg which you can use instead of this `__doc__` mambo-jambo.